src/pages/quiz/creation/MultipleChoiceQuestionView.tsx

```diff
@@ -0,0 +1,152 @@
+                              d="M42.42,12.401c0.774-0.774,0.774-2.028,0-2.802L38.401,5.58c-0.774-0.774-2.028-0.774-2.802,0	L24,17.179L12.401,5.58c-0.774-0.774-2.028-0.774-2.802,0L5.58,9.599c-0.774,0.774-0.774,2.028,0,2.802L17.179,24L5.58,35.599	c-0.774,0.774-0.774,2.028,0,2.802l4.019,4.019c0.774,0.774,2.028,0.774,2.802,0L42.42,12.401z"></path>
+                        <linearGradient id="hbE9Evnj3wAjjA2RX0We2b_OZuepOQd0omj_gr2" x1="27.373" x2="40.507" y1="27.373"
+                                        y2="40.507" gradientUnits="userSpaceOnUse">
+                            <stop offset="0" stopColor="#a8142e"></stop>
+                            <stop offset=".179" stopColor="#ba1632"></stop>
+                            <stop offset=".243" stopColor="#c21734"></stop>
+                        </linearGradient>
+                        <path fill="url(#hbE9Evnj3wAjjA2RX0We2b_OZuepOQd0omj_gr2)"
+                              d="M24,30.821L35.599,42.42c0.774,0.774,2.028,0.774,2.802,0l4.019-4.019	c0.774-0.774,0.774-2.028,0-2.802L30.821,24L24,30.821z"></path>
+                    </svg>
+                    <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="0 0 48 48"
+                         style={
+                             {
+                                 marginRight: !isEditable && question.answer[index] && question.answer[index] != userAnswer[index] ? 20 : 0,
+                                 marginLeft: !isEditable && question.answer[index] && question.answer[index] != userAnswer[index] ? 20 : 0,
+                                 width: !isEditable && question.answer[index] && question.answer[index] != userAnswer[index] ? 20 : 0,
+                                 height: !isEditable && question.answer[index] && question.answer[index] != userAnswer[index] ? 20 : 0
+                             }
+                         }
+                         visibility={!isEditable && question.answer[index] != userAnswer[index] ? 'visible' : 'hidden'}>
+                        <path
+                            d="M18,43c-0.256,0-0.512-0.098-0.707-0.293l-16-16C1.105,26.52,1,26.265,1,26v-2c0-0.347,0.18-0.668,0.474-0.851	c0.295-0.182,0.663-0.198,0.973-0.044l15.466,7.733l26.511-18.656c0.307-0.215,0.707-0.241,1.037-0.07	C45.792,12.285,46,12.626,46,13v2c0,0.265-0.105,0.52-0.293,0.707l-27,27C18.512,42.902,18.256,43,18,43z"></path>
+                        <polygon fill="#fff" points="45,13 40,8 18,30 7,19 2,24 18,40"></polygon>
+                        <path
+                            d="M18,41c-0.256,0-0.512-0.098-0.707-0.293l-16-16c-0.391-0.391-0.391-1.023,0-1.414l5-5c0.391-0.391,1.023-0.391,1.414,0	L18,28.586L39.293,7.293c0.391-0.391,1.023-0.391,1.414,0l5,5c0.391,0.391,0.391,1.023,0,1.414l-27,27	C18.512,40.902,18.256,41,18,41z M3.414,24L18,38.586L43.586,13L40,9.414L18.707,30.707c-0.391,0.391-1.023,0.391-1.414,0L7,20.414	L3.414,24z"></path>
+                    </svg>
+                    <BouncyCheckbox
```

Comment:
Jeśli dałbyś radę to fajnie byłoby żeby te checkbox'y były kwadratowe (bo to pytanie wielokrotnego wyboru)

Reply:
narazie zostawmy te, ale zgadzam się, potem będą do zmiany